pkg/api/api.go

```diff
@@ -68,13 +68,13 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/configuration", reqGrafanaAdmin, hs.Index)
 	r.Get("/admin", reqGrafanaAdmin, hs.Index)
 	r.Get("/admin/settings", reqGrafanaAdmin, hs.Index)
-	r.Get("/admin/users", reqGrafanaAdmin, hs.Index)
-	r.Get("/admin/users/create", reqGrafanaAdmin, hs.Index)
-	r.Get("/admin/users/edit/:id", reqGrafanaAdmin, hs.Index)
+	r.Get("/admin/users", authorize(reqGrafanaAdmin, accesscontrol.ActionUsersRead, accesscontrol.ScopeUsersAll), hs.Index)
```

Comment:
The `authorize` middleware isn't written to handle browser-style requests, iirc there's a few differences in how we handle browsers and API clients in the `req*` family of middlewares. Not urgent, but we'll need to fix that.